src/dom/DomEvent.DoubleTap.js

```diff
@@ -9,6 +9,19 @@ var _touchstart = Browser.msPointer ? 'MSPointerDown' : Browser.pointer ? 'point
 var _touchend = Browser.msPointer ? 'MSPointerUp' : Browser.pointer ? 'pointerup' : 'touchend';
 var _pre = '_leaflet_';
 
+function browserFiresNativeDblClick(e) {
+	// See https://github.com/w3c/pointerevents/issues/171
+	// Note however, that Chrome stopped firing native dblclick for
+	// touch since that issue was written.
+	return (Browser.ie || e.pointerType === 'mouse');
+}
+
+function browserNeedsAManualDblClick(e) {
+	var oe = e.originalEvent;
+
+	return (L.Browser.Touch && (oe && oe.sourceCapabilities.firesTouchEvents));

```

Comment:
@sashadev-sky, I've failed to understand purpose of `sourceCapabilities.firesTouchEvents` here.

[AFAIK](https://developer.mozilla.org/en-US/docs/Web/API/InputDeviceCapabilities_API), this can be useful when we listen to both touch and mouse events.
But for our code it is not in case, as there pointer events only (and we have exact `pointerType` value for them).

And what is mentioned 'level of stability'?
> It also introduces a level of stability. For ex, if a newer version of Chrome does start to fire native dblclick events, the functionality will not break